### DISCUSSION TITLE
'An internal error occurred in your connection.' - while changing worlds

### DISCUSSION THREAD

**Discussion by RutekGutek** [PrismarineJS/mineflayer#3528] [Q&A]
I was trying to make a bot for specific minecraft server but I got an error.
Every time after the bot logs in with the login command ( bot.chat('/login password'); ) It gets kicked instantly (Normaly after logging in, server should transfer you to lobby (different world))

Code:

```
const mineflayer = require('mineflayer');

const bot = mineflayer.createBot({
    host: 'ip',  
    port: 25565,        
    username: 'name',
    version: '1.21.2'  
});

bot.on("kicked", (r) => {
    console.log(r);
});

bot.on('messagestr', (message) => {
    console.log(message);
    if (message.includes('Verified')) {
        bot.chat('/login password');
}
});
```

Kick reason:

![image](https://github.com/user-attachments/assets/2da26d7f-3c0b-43ac-8d59-f2ec9da86150)

Do you have any ideas?

**Comment by wfogame** [comment]
Velocity doesnt work well this, im too late lLMAO, its cuz velcotiy has known bugs
